src/options.cpp

```diff
@@ -1671,6 +1671,17 @@ void options_manager::add_options_graphics()
 
     get_option( "TILES" ).setPrerequisite( "USE_TILES" );
 
+    mOptionsSort["graphics"]++;
+
+    add( "MEMORY_MAP_MODE", "graphics", translate_marker( "Memory map drawing mode" ),
+    translate_marker( "Specified the mode in which the memory map is drawn.  Requires restart." ), {
+        { "color_pixel_darken", translate_marker( "Classic (Darkened)" ) },
+        { "color_pixel_sepia", translate_marker( "Modern (Sepia)" ) }
```

Comment:
I don't think both "Classic" and "Modern" names are really needed here. What if someone adds another drawing mode? Call it "Super-Modern" or "Neo-Modern"? Simple "Darkened" and "Sepia" is enough.